backend/src/controllers/serviceControllers.js

```diff
@@ -0,0 +1,46 @@
+import githubService from '../thirdParties/github/githubService';
+
+const getAppropriateService = serviceName => {
+  if (serviceName === 'github') {
+    return githubService;
+  }
+  return githubService;
+};
+
+const getServices = async (req, res) => {
```

Comment:
async function without await?

Try/ catch makes no sense here. There's an assignment and return. Which one do you think would throw ?